lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -1335,6 +1338,7 @@ impl X64FunctionCode {
             fsm.call_offsets.insert(
                 offset,
                 OffsetInfo {
+                    end_offset: offset + 1,
```

Comment:
Is this a bug-fix? If so, could you pull it out into its own PR for easier review both of its own change and the OSR patch?

Reply:
This is a required change for OSR to work for LLVM, since we cannot determine the exact address of the trapping instruction (IR is more high-level).